src/home_robot_hw/home_robot_hw/ros/recorder.py

```diff
@@ -53,6 +53,7 @@ def finish_recording(self) -> int:
         print(f"... done recording trial named: {filename}.")
         self.idx += 1
         print("Ready for next episode...Press START to begin ")
+        self.robot.switch_to_navigation_mode()
```

Comment:
why?

Reply:
So that the teleoperator can move the robot around in-between data recording sessions. Based on Jay's feedback.